### DISCUSSION TITLE
What is native_menu and how can i disable / enable it ?

### DISCUSSION THREAD

**Discussion by RaphaelAsla** [hrsh7th/nvim-cmp#1071] [Q&A]
what title says.

**Comment by fitrh** [comment]
It is the window appearance, by default, cmp using `custom` view, which is a floating window.
The `native` view uses vim bulit-in completion window and AFAIK, is less styleable.

You can use `view` key


```lua
cmp.setup({
    -- ...
    view = 'native' -- or 'custom' or 'wildmenu'
    -- ...
})
```

**Comment by thjbdvlt** [comment]
I think that it changed to:

```lua
cmp.setup({
  -- ...
  view = {entries = {name = "native"}},
  -- ...
})
